boards/particle_boron/Cargo.toml

```diff
@@ -4,7 +4,7 @@
 
 [package]
 name = "particle_boron"
-version.workspace = true
+version = "0.2.0"
```

Comment:
We don't typically increment the version number of boards. I think that maintaining different versions for each board is confusing, when we then also have our canonical Tock release versions (which cover the entire repository including all boards).

Reply:
I see, dropping this commit then.